rules-tests/CodeQuality/Rector/FuncCall/SingleInArrayToCompareRector/Fixture/fixture.php.inc

```diff
@@ -8,8 +8,6 @@ class Fixture
     {
         $isIt = in_array(strtolower($type), ['$this'], true);
         $isIt = in_array(strtolower($type), ['$this']);
-
-        $isIt = in_array(strtolower($type), ['$this', 'two']);
```

Comment:
this skipped part for multi data moved to specific fixture

https://github.com/rectorphp/rector-src/pull/5205/files#diff-a723d5612fbb34df879841e8ee175b22f5f9cd3d1523111e41d52ac3d74aa27f

to ease discover bug in the future.